server/tests/integration/http/statutCandidatsRoute-test.js

```diff
@@ -176,4 +176,23 @@ httpTests(__filename, ({ startServer }) => {
     assert.deepStrictEqual(response.status, 413);
     assert.notDeepStrictEqual(await StatutCandidat.countDocuments({}), nbItemsToTest);
   });
+
+  it.only("Vérifie que la route statut-candidats/test fonctionne avec une bonne clé d'API", async () => {
+    const { httpClient } = await startServer();
```

Comment:
tu dois ajouter la création d'un user de l'API car la route est sécurisée pour des user ayant été créés .

```suggestion
// Create & check api user
    const userApiCreated = await createApiUser();
    assert.deepStrictEqual(userApiCreated.username, "userApi");
    assert.deepStrictEqual(userApiCreated.permissions.length > 0, true);
    assert.deepStrictEqual(userApiCreated.apiKey, goodApiKey);
```

Reply:
Création d'user needed